src/extension_consts.ml

```diff
@@ -32,6 +32,20 @@ module Commands = struct
   let evaluate_selection = ocaml_prefixed "evaluate-selection"
 
   let open_repl = ocaml_prefixed "open-repl"
+
+  let next_hole = ocaml_prefixed "next-hole"
+end
+
+module Command_errors = struct
+  let avail_when_editor_open ~expl cmd_name =
```

Comment:
You might as well make this `available_when_editor_open`.

Omitting 4 characters doesn't help much with brevity here, instead it makes it harder to read.